rollbar_dart/lib/src/sender/http_sender.dart

```diff
@@ -52,6 +59,8 @@ class HttpSender implements Sender {
 
       return true;
     } catch (error, stackTrace) {
+      if (!_State.suspended) _State.suspend(30.seconds);
+
       log('Exception sending payload',
           time: DateTime.now(),
           level: Level.error.value,
```

Comment:
return false; ?

Reply:
At the end of the catch clause. Just below the log.